pkg/cli/cliflags/flags.go

```diff
@@ -256,7 +256,20 @@ Execute the SQL statement(s) on the command line, then exit. This flag may be
 specified multiple times and each value may contain multiple semicolon
 separated statements. If an error occurs in any statement, the command exits
 with a non-zero status code and further statements are not executed. The
-results of each SQL statement are printed on the standard output.`,
+results of each SQL statement are printed on the standard output.
+
+This flag is incompatible with --input-file / -f.`,
```

Comment:
I don't have an alternative, but is `input-file` used in other CLIs? The -f feels familiar+intuitive, the --input-file doesn't as much (partly because it doesn't start with an f).